build/integration.sh

```diff
@@ -26,7 +26,8 @@ printf "" # Refresh sudo credentials if necessary.
 function start {
   set +e  # We want to handle errors if cAdvisor crashes.
   echo ">> starting cAdvisor locally"
-  GORACE="halt_on_error=1" ./cadvisor --docker_env_metadata_whitelist=TEST_VAR --v=6 --logtostderr $CADVISOR_ARGS &> "$log_file"
+  # This cpuset, percpu, memory, disk, diskIO, network, perf_event metrics should be enabled.
```

Comment:
Now that -enable_metrics is used, this could say e.g. "enable only metrics required by the test", or be even removed.

Reply:
In my opinion, that comment is more accurate than `"enable only metrics required by the test"`.

Reply:
Comment metrics list just duplicates -enable_metrics argument, so it's redundant / can get out of sync later. Comment is a bit weirdly at the moment too ("This" -> "The"?).  But feel free to ignore my comment "bike-shedding", it's not really important, like rest of this PR.